dhall-toml/tasty/data/empty.dhall

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
We should probably focus on converting Dhall _values_ first. ;)

```suggestion
{=}
```

Reply:
That's a good idea! I've only understood the difference as I've started digging into the internals